kernel/syscalls/clone3.rs

```diff
@@ -0,0 +1,139 @@
+use crate::prelude::*;
+use crate::syscalls::SyscallHandler;
+use crate::{
+    ctypes::*
+};
+use kerla_runtime::address::UserVAddr;
+use bitflags::bitflags;
+
+
+bitflags! {
+    /// Flags used by [`clone()`] system call
+    /// Source: `/usr/include/linux/sched.h`
+    ///
+    /// [`clone()`]: https://linux.die.net/man/2/clone
+    pub struct CloneFlags : c_uint {
+        const CLONE_VM             = 0x00000100;     // set if VM shared between processes
```

Comment:
Please avoid adding a comment at the end of the line and add a full stop at the end of the comment.

```suggestion
        /// Set if VM shared between processes.
        const CLONE_VM             = 0x00000100;     
```